[VirusTotal] Add private scanning

pabloperezj opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

samuelFain commented on Packs/VirusTotal/Integrations/VirusTotalV3/README.md:
````diff
@@ -2158,3 +2158,384 @@ Retrieves result of the last Sigma analysis.
+
+#### Command Example
+
+```!vt-privatescanning-file file=f912398cb3542ab704fe917af4a60d4feee21ac577535b10453170f10c6fd6de```
````
Please replace any file hash, file path or any other possible sensitive value in this file with a 'dummy value' (`file=exmaple-file-hash` for example).
You can use `demisto-sdk secrets` command to help you locate all possible sensitive values in the file ([more information here](https://xsoar.pan.dev/docs/concepts/demisto-sdk#secrets)).

samuelFain commented on .devcontainer/devcontainer.json:
```diff
@@ -1,4 +1,4 @@
-// Development container for Content. Poetry hash 1.11.0
+// Development container for Content. Poetry hash 4a7e3418ab85fe8465677dd449b5a2fd883d0e2e4c008e469f0a2b658c341216
```
Please revert back to:
```suggestion
// Development container for Content. Poetry hash 1.11.0
```

samuelFain reviewed (commented):
Hi @pabloperezj, Thank you for contributing!
Added a few comments, please review them.
Also, @erezh31 from our security team will review the playbook related changes shortly.
Feel free to contact me in the meanwhile for any questions/clarifications.

pabloperezj commented:
> Hi @pabloperezj, and thank you for contributing! Added a few comments, please review them. Also, @erezh31 from our security team will review the playbook related changes shortly. Feel free to contact me in the meanwhile for any questions/clarifications.

Hi @samuelFain, thanks for the comments. The hashes added were random, so did not contains sensitive information, but I fixed it anyway. Thank you very much! I hope everything is fine now!

efelmandar reviewed (dismissed):
Hi @pabloperezj thank you very much for your contribution! I reviewed the playbook in your PR and found some points which I think can be improved,

- `File` Playbook input - it's better to use `${File.EntryID}` to avoid objects without EntryID which is needed the get the actual file
- In task `Is there a file to detonate?` (no. 2) - Check for the `Inputs.File` instead of the `File`
- In task `Set Context` (no. 5) - should pass the key name without `${}`

pabloperezj commented:
> Hi @pabloperezj thank you very much for your contribution! I reviewed the playbook in your PR and found some points which I think can be improved,
> 
> * `File` Playbook input - it's better to use `${File.EntryID}` to avoid objects without EntryID which is needed the get the actual file
> * In task `Is there a file to detonate?` (no. 2) - Check for the `Inputs.File` instead of the `File`
> * In task `Set Context` (no. 5) - should pass the key name without `${}`

Hi @efelmandar, done! Thank you very much!

samuelFain commented:
@pabloperezj Great work!
Could you please update the docker image (see `Run Validations` check) so that we can pass all the build checks? (Would do it myself but I dont have permissions to push commits to this PR)

pabloperezj commented:
> @pabloperezj Great work! Could you please update the docker image (see `Run Validations` check) so that we can pass all the build checks? (Would do it myself but I dont have permissions to push commits to this PR)

Done! Thank you @samuelFain!

samuelFain commented:
@pabloperezj, We noticed the forked repo is named with `cortex-xsoar` instead of the suggested `content` format.
This current situation is preventing us to continue with the merging process.
Could you please manually rename the repo name using this manual?
https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository
Please notify me if there are any questions.
Thanks!

pabloperezj commented:
> @pabloperezj, We noticed the forked repo is named with `cortex-xsoar` instead of the suggested `content` format. This current situation is preventing us to continue with the merging process. Could you please manually rename the repo name using this manual? https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository Please notify me if there are any questions. Thanks!

@samuelFain done! Thanks!
